### ISSUE TITLE
Binaries are not downloaded

### ISSUE THREAD

**Issue by markknol** [HaxeFoundation/npm-haxe#18]
I just tried haxe-npm but it does not seem to download the actual binaries of Haxe. This leads to 
```
#npm install haxe -g
npm WARN deprecated gulp-util@3.0.8: gulp-util is deprecated - replace it, following the guidelines at https://medium.com/gulpjs/gulp-util-ca3b1f9f9ac5
/usr/bin/haxe -> /usr/lib/node_modules/haxe/bin/haxe
/usr/bin/haxelib -> /usr/lib/node_modules/haxe/bin/haxelib
/usr/bin/neko -> /usr/lib/node_modules/haxe/bin/neko
> haxe@4.1.4 install /usr/lib/node_modules/haxe
> node install.js
no dependencies
clean folder
Installing Haxelib Dependencies to /root
+ haxe@4.1.4
added 283 packages from 129 contributors in 6.617s
```

```
# haxe --help
{ Error: spawn /usr/lib/node_modules/haxe/downloads/haxe/haxe ENOENT
    at exports._errnoException (util.js:1020:11)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:197:32)
    at onErrorNT (internal/child_process.js:376:16)
    at _combinedTickCallback (internal/process/next_tick.js:80:11)
    at process._tickCallback (internal/process/next_tick.js:104:9)
    at Module.runMain (module.js:606:11)
    at run (bootstrap_node.js:383:7)
    at startup (bootstrap_node.js:149:9)
    at bootstrap_node.js:496:3
  code: 'ENOENT',
  errno: 'ENOENT',
  syscall: 'spawn /usr/lib/node_modules/haxe/downloads/haxe/haxe',
  path: '/usr/lib/node_modules/haxe/downloads/haxe/haxe',
  spawnargs: [ '--help' ] }
```

**Comment by dmouton** [comment]
I know and it should be fixed with the 4.1.7

https://github.com/HaxeFoundation/npm-haxe/blob/feature/haxelib_dependencies/package.json#L3

**Comment by markknol** [comment]
what does haxe@beta mean today? Haxe 4 or Haxe 3.4.7?

**Comment by damoebius** [comment]
It's the latest npm module on the beta channel. The default haxe version is 3.4.7.
You can specify another in your package.json

**Comment by markknol** [comment]
Can you help how to install this? I have not much experience with npm, trying to get my head around it.
- I created an empty `{}` package.json. 
- I run `npm install haxe --save-dev`
- Error:
```
> haxe@4.1.8 install C:\Users\mark\Desktop\npm-haxe-test\node_modules\haxe
> node install.js

module.js:474
    throw err;
    ^

Error: Cannot find module 'download-status'
    at Function.Module._resolveFilename (module.js:472:15)
    at Function.Module._load (module.js:420:25)
    at Module.require (module.js:500:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (C:\Users\mark\Desktop\npm-haxe-test\node_modules\haxe\lib\cache.js:5:22)
    at Module._compile (module.js:573:32)
    at Object.Module._extensions..js (module.js:582:10)
    at Module.load (module.js:490:32)
    at tryModuleLoad (module.js:449:12)
    at Function.Module._load (module.js:441:3)
npm WARN test2 No description
npm WARN test2 No repository field.
npm WARN test2 No license field.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! haxe@4.1.8 install: `node install.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the haxe@4.1.8 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\mark\AppData\Roaming\npm-cache\_logs\2018-04-05T08_24_10_732Z-debug.log
```
- Ok I need `4.1.10`, like you said.
- I run `npm install haxe@4.1.10 --save-dev`
- Error:
```
npm ERR! code ETARGET
npm ERR! notarget No matching version found for haxe@4.1.10
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\mark\AppData\Roaming\npm-cache\_logs\2018-04-05T08_25_45_426Z-debug.log
```

**Comment by damoebius** [comment]
Try again :)
npm install haxe@4.1.10 or npm install haxe@beta

**Comment by jgranick** [comment]
I have older projects that use `"haxe": "^4.1.0",`, and these are currently broken for some reason. Installing `npm install haxe@beta` manually appears to fix it, but before I was getting errors about "download-status" being missing, or other issues

**Comment by damoebius** [comment]
My bad. I mistakenly released the 4.1.8 on the main channel.
So i just deprecated it.
Sorry for the inconvenience
